code/modules/surgery/organs/eyes.dm

```diff
@@ -61,14 +61,17 @@
 /obj/item/organ/eyes/night_vision/alien
 	name = "alien eyes"
 	desc = "It turned out they had them after all!"
-	see_in_dark = 8
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
```

Comment:
Why this change?

Reply:
Duplicated lines, it inherits that already